CHANGELOG.md

```diff
@@ -27,6 +27,12 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 
 ---
 
+## [9.20] 🎁 Unwrapped Icons
```

Comment:
could technically be a patch version, but using a judgement call here to create a more notable milestone in case this breaks anything (just in case)